Google.PowerShell.DotnetCore/Google.PowerShell.DotnetCore.csproj

```diff
@@ -0,0 +1,49 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Google Cloud Tools for PowerShell, targeting the .NET Core runtime</Description>
+    <Copyright>Google Inc. 2016</Copyright>
+    <AssemblyTitle>Google Cloud Tools for PowerShell</AssemblyTitle>
+    <TargetFramework>netstandard1.6</TargetFramework>
```

Comment:
I'm curious why only netstandard1.6? I'm also not sure if this actually matters to end user like a client lib does.

Reply:
It's the one that I tested with at that time. I try this module with the current PowerShell Core (which uses .NET standard 2.0) and it works just fine (since it's supposed to be compatible).